style/base/base.scss

```diff
@@ -6,11 +6,17 @@ html, body {
 body {
     overflow-y: scroll;
     & > footer {
-        margin: 100px 0 20px 0;
-        text-align: center;
-        & > a {
-            display: inline-block;
-            padding: 10px;
+        display: flex;
+        justify-content: space-around;
+        margin: 100px auto 20px auto;
+
+        ul {
+          list-style: none;
+          padding: 0px;
+
+          li {
+            margin: 10px auto;
+          }
```

Comment:
You have inconsistent indentation here, 2 spaces instead of 4 in the rest of the file.